workspaceitem-data-sherpa-policy.md

```diff
@@ -0,0 +1,114 @@
+              "option" : 1,
+              "conditions" : [ "Publisher copyright and source must be acknowledged", "Must link to publisher version" ],
+              "prerequisites" : [ ],
+              "locations" : [ "Author's Homepage", "Institutional Website" ],
+              "licenses" : [ "CC BY-NC-ND" ],
+              "embargo" : null
+            }, {
+              "articleVersion" : "accepted",
+              "option" : 2,
+              "conditions" : [ "Publisher copyright and source must be acknowledged", "Must link to publisher version" ],
+              "prerequisites" : [ "If Required by Funder" ],
+              "locations" : [ "Non-Commercial Repository" ],
+              "licenses" : [ "CC BY-NC-ND" ],
+              "embargo" : null
+            }, {
+              "articleVersion" : "accepted",
+              "option" : 3,
+              "conditions" : [ "Publisher copyright and source must be acknowledged", "Must link to publisher version" ],
+              "prerequisites" : [ ],
+              "locations" : [ "Non-Commercial Repository" ],
+              "licenses" : [ ],
+              "embargo" : null
+            } ],
+            "urls" : {
+              "http://download.thelancet.com/flatcontentassets/authors/lancet-information-for-authors.pdf" : "Guidelines for Authors",
+              "http://www.thelancet.com/journals/lancet/article/PIIS0140-6736%2813%2960720-5/fulltext" : "The Lancet journals welcome a new open access policy",
+              "http://www.thelancet.com/lancet-information-for-authors/after-publication" : "What happens after publication?",
+              "http://www.thelancet.com/lancet/information-for-authors/disclosure-of-results" : "Disclosure of results before publication",
+              "https://www.elsevier.com/__data/assets/pdf_file/0005/78476/external-embargo-list.pdf" : "Journal Embargo Period List",
+              "https://www.elsevier.com/__data/assets/pdf_file/0011/78473/UK-Embargo-Periods.pdf" : "Journal Embargo List for UK Authors"
+            },
+            "openAccessProhibited" : false,
+            "publicationCount" : 0,
+            "preArchiving" : "can",
+            "postArchiving" : "can",
+            "pubArchiving" : "cannot"
+          } ],
+          "inDOAJ" : false
+        } ]
+      }
+    }
+```
+
+Please note that the sherpaResponse attribute will be never null but could eventually contains only the isError flag set to false when nothing is found on Sherpa.
+The section data will be not included at all if the item has not information (i.e. issns) suitable to query Sherpa/RoMEO.
+
+## Patch operations
+The PATCH method expects a JSON body according to the [JSON Patch specification RFC6902](https://tools.ietf.org/html/rfc6902)
+
+### Remove
+This section is almost readonly. To limit the load on the Sherpa webservices and improve performance sherpa data are cached. It is possible to force a fresh call removing the retrievalTime from the section as follow
```

Comment:
This description is confusing.  I'd recommend we update it to say something like:
```
### Remove /retrievalTime

To limit the load on Sherpa webservices and improve performance, all Sherpa data is cached.  You can force a refresh by removing the `retrievalTime` as follows:
```

(If we can provide details here on how frequently the cache will automatically be refreshed, that'd also be beneficial to document).